boards/fruitbar/main.py

```diff
@@ -0,0 +1,284 @@
+import board
+
+from kb import KMKKeyboard
+
+from kmk.extensions.peg_oled_Display import (
+    Oled,
+    OledData,
+    OledDisplayMode,
+    OledReactionType,
+)
+from kmk.keys import KC
+from kmk.modules.layers import Layers
+from kmk.scanners import DiodeOrientation
+
+keyboard = KMKKeyboard()
+
+keyboard.SCL = board.GP27
+keyboard.SDA = board.GP26
+
```

Comment:
Why is this not part of the keyboard class?

Reply:
part of the process until I got it to work.  yeeting now.